contracts/README.md

```diff
@@ -15,31 +15,76 @@ $ npm install @chainlink/contracts --save
 @chainlink/contracts
 ├── src # Solidity contracts
 │   └── v0.8
-└── abi # ABI json output
-    └── v0.8
+│── abi # ABI json output
+│   └── v0.8
+└── gas-snapshots # gas metered results of available non-fuzz tests
```

Comment:
This readme will be the NPM readme and the gas snapshot isn't included in the NPM release